### ISSUE TITLE
Double scrollbar issue in Chrome

### ISSUE THREAD

**Issue by riklewis** [balzss/luxbar#5]
I've not tested other browsers, but certainly in Chrome there's a double scrollbar appearing when the responsive/hamburger menu drops down.  

I've recorded a video to show this, but Github doesn't like MP4 files, so here's a link... [Luxbar.mp4](www.rik.onl/files/Luxbar.mp4)

**Comment by balzss** [comment]
The file is not found, could you please upload it again? Also, what OS are you using?

**Comment by Pustur** [comment]
@balzss [working link](http://www.rik.onl/files/Luxbar.mp4)

**Comment by riklewis** [comment]
Sorry about that, missed the http:// off the beginning, fixed now.  And it's Windows 7, Chrome 55.0.2883.87.

**Comment by balzss** [comment]
Seems like it's only appearing in Windows. I'll try to get a windows machine and resolve the problem as soon as I can.

**Comment by riklewis** [comment]
Great.  Awesome work, by the way!

**Comment by riklewis** [comment]
I believe if you remove "overflow: auto" from line 189 in luxbar.css, that resolves the problem.  I don't know if creates a problem on non-Windows systems or not though.

**Comment by skowie** [comment]
Appears in all mayor browsers on Windows 10 for me. But removing "overflow: auto" as suggested by riklewis seems to fix it.

Checked it in Chrome, Firefox, Opera, IE and Edge on Win 10.

**Comment by balzss** [comment]
I think the problem with that solution was that it removed the opening animation completely but I have to double check it.

**Comment by AKTheKnight** [comment]
~~On Chrome 60.0.3112.101 I cannot reproduce this bug, is anyone else able to?~~

Scratch that, I am still able to reproduce it

**Comment by riklewis** [comment]
It's still happening for me on Windows 10/Chrome 60.0.3112.113 (Official Build) (64-bit).

**Comment by balzss** [comment]
I'm thinking that [this](https://stackoverflow.com/a/30680994/4503695) might solve it for chrome but probably not for firefox. Can someone with Windows confirm that the bug is reproducible with Firefox as well?

**Comment by AKTheKnight** [comment]
Yep it happens in Firefox. The second scrollbar appears and then goes again, same as on Chrome
